platform/mellanox/mlnx-platform-api/sonic_platform/pcie.py

```diff
@@ -30,23 +30,56 @@
 
 SYSFS_PCI_DEVICE_PATH = '/sys/bus/pci/devices/'
 
+# Constants from module_base.py
+PCIE_DETACH_INFO_TABLE = "PCIE_DETACH_INFO"
+PCIE_OPERATION_DETACHING = "detaching"
+BULEFIELD_SOC_ID = "c2d5"
+BLUEFIELD_CONNECTX_ID = "a2dc"
+
 
 class Pcie(PcieUtil):
     # check the current PCIe device with config file and return the result
     # use bus from _device_id_to_bus_map instead of from yaml file
     def get_pcie_check(self):
         self.load_config_file()
+        return_confInfo = []
```

Comment:
Should we also check the vendor ID here? The chance of a coincidence is minimal, but still, it presents